frontend/services/entities/software.ts

```diff
@@ -205,6 +296,33 @@ const handleEditPackageForm = (
   }
 };
 
+const handleDisplayNameAppStoreAppForm = (
+  formData: ISoftwareDisplayNameFormData,
+  body: IEditAppStoreAppPostBody
+) => {
+  body.display_name = formData.displayName || "";
```

Comment:
This is already accounting for VPP + Google Play store apps in a subsequent PR and should remove code from mdm_apple.ts if conflict arises as this replaces that code.